package.json

```diff
@@ -26,7 +26,7 @@
     "test:smokeTest": "webdriver-manager update --versions.chrome 2.40 && protractor test/config/conf.js --cucumberOpts.tags='@smoke and not @bug and not @broken'",
     "test:crossbrowser": "protractor test/config/crossbrowser.conf.js",
     "test:functional": "./bin/runFunctionalTests.sh",
-    "test:functionalTests": "webdriver-manager update --versions.chrome 2.40 && protractor test/config/conf.js --cucumberOpts.tags='@functional and not @bug and not @broken'",
+    "test:functionalTests": "webdriver-manager update --versions.chrome 2.40 && protractor test/config/conf.js --cucumberOpts.tags='@mediaviewer and not @bug and not @broken'",
```

Comment:
Just realised this line will disable @functional tests and enable only @mediaviewer ones. We need to change the cucumberOpts.tags to include @functional.

Reply:
Yes of course - this was just an investigation bit into whether mediaviewer tests would pass as they were commented out previously.